src/Libraries/RevitNodes/Elements/AdaptiveComponent.cs

```diff
@@ -74,6 +191,78 @@ private static void UpdatePlacementPoints(Autodesk.Revit.DB.FamilyInstance fi, X
             }
         }
 
+        /// <summary>
+        /// Set the family symbol for the internal family instance 
+        /// </summary>
+        /// <param name="ft"></param>
+        private void InternalSetFamilySymbol(FamilyType ft)
+        {
+            TransactionManager.Instance.EnsureInTransaction(Document);
+
+            InternalFamilyInstance.Symbol = ft.InternalFamilySymbol;
+
+            TransactionManager.Instance.TransactionTaskDone();
+
+        }
+
+        /// <summary>
+        /// Set the positions of the InternalFamilyInstace from an array of uvs
+        /// </summary>
+        /// <param name="uvs"></param>
+        /// <param name="faceReference"></param>
+        private void InternalSetUvsAndFace(Autodesk.Revit.DB.UV[] uvs, Autodesk.Revit.DB.Reference faceReference)
+        {
+            TransactionManager.Instance.EnsureInTransaction(Document);
+
+            var placePointIds = AdaptiveComponentInstanceUtils.GetInstancePlacementPointElementRefIds(InternalFamilyInstance);
+
+            if (placePointIds.Count() != uvs.Length)
```

Comment:
This is a private method. If the number of uv parameters does not equal to the number of placement points, an exception will be thrown.